exercises/bracket-push/test/Tests.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import BracketPush (isPaired)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "bracket-push" $
+          describe "isPaired" $ for_ cases test
+  where
+    test Case{..} = it description $ isPaired input `shouldBe` expected
+
+-- Test cases adapted from
+-- `exercism/x-common/exercises/bracket-push/canonical-data.json`
+-- on 2017-03-29.
```

Comment:
same q as https://github.com/exercism/xhaskell/pull/511#discussion_r109295131 - how should we put the version in here?